docs/concepts/alert_activations.ipynb

```diff
@@ -0,0 +1,550 @@
+{
```

Comment:
<p>comment: <code># list all alert activations for all projects</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/6881/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
@TomerShor it is for default project, default value of project is `default`

Reply:
Oh right. I suggested to add a comment to align to what you did in the other sdk options below.